src/pages/loans/Loans.js

```diff
@@ -0,0 +1,48 @@
+// import styled from 'styled-components/macro'
+import { useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import { loadFromLocal } from '../../helper/localStorage'
+import { BASE_URL } from '../../helper/url'
+import Showloans from '../../components/Showloans/Showloans'
+
+Loans.propTypes = {
+  token: PropTypes.string,
+}
+
+export default function Loans() {
+  const [availableLoans, setAvailableLoans] = useState([])
+  const { token } = loadFromLocal('token')
+
+  useEffect(() => {
+    fetch(BASE_URL + '/types/loans', {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${token}`,
+      },
+    })
+      .then(res => res.json())
```

Comment:
You could extract the fetch to a file `services/getLoans.js`.

```suggestion

  useEffect(() => {
   getLoans()
```